docs/TESTS.md

```diff
@@ -20,9 +20,27 @@ Once you have been able to compile the code it will create a DLL in the ```\bin\
 The NUnit runner will automatically reload the DLL if it has been updated.
 
 ### Mac
-Xamarin Studio also ships with NUnit. From the new project dialog, just select an NUnit class library.
+Xamarin Studio also ships with NUnit. To set the tests up you will have to add NUnit as a new project to your solution, name it correctly and set a reference to your solution.
```

Comment:
Perhaps "add NUnit as a new project to your solution" could be rephrased to something like " add an NUnit library project to your solution"? As you are not actually adding NUnit as a new project, but just use the template.